examples/voice_agents/langfuse_trace.py

```diff
@@ -42,6 +42,17 @@ def setup_langfuse(
     trace_provider.add_span_processor(BatchSpanProcessor(OTLPSpanExporter()))
     set_tracer_provider(trace_provider)
 
+    # (optional) enable OpenAI OpenTelemetry instrumentation
```

Comment:
you may want to remove this comment. if llm_request is the generation span, then adding both will most likely result in double counting generation calls, tokens, etc in most observability platforms.

Reply:
you can see the double counting here. should i be able to see this trace? is it public?
<img width="486" height="129" alt="Screenshot 2025-07-15 at 11 06 26 AM" src="https://github.com/user-attachments/assets/6d2fedd3-a3c2-4973-b92f-c0cf0b1bdcd0" />

Reply:
it looks like it's double counting the price. not sure why the token usage is different, maybe a livekit calculation issue or something with langfuse?